multicluster/controllers/multicluster/serviceexport_controller.go

```diff
@@ -160,31 +162,22 @@ func (r *ServiceExportReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 		klog.InfoS("Skip reconciling, no corresponding ServiceExport")
 		return ctrl.Result{}, nil
 	}
-
-	if *r.remoteCommonAreaManager == nil {
-		klog.InfoS("ClusterSet has not been initialized properly, no remote cluster manager")
+	requeue, remoteCluster, err := r.checkRemoteManager()
```

Comment:
remoteCluster -> commonArea to be clear? Do we need "remote" in the name?

Reply:
done, rename it to commonArea